docs/howtos/customizations/metrics/_train_your_own_metric.md

```diff
@@ -0,0 +1,156 @@
+from ragas.embeddings import LangchainEmbeddingsWrapper
+from langchain_openai import ChatOpenAI
+from langchain_openai import OpenAIEmbeddings
+llm = LangchainLLMWrapper(ChatOpenAI(model="gpt-4o-mini"))
+embeddings = LangchainEmbeddingsWrapper(OpenAIEmbeddings())
+```
+
+## Evaluation
+
+### Load sample evaluation dataset
+Here, we are loading the sample dataset for evaluation. You can replace it with your own dataset. 
+
+
+```python
+dataset = load_dataset("explodinggradients/ELI5",split="test")
+eval_dataset = EvaluationDataset.from_hf_dataset(dataset)
+```
+
+### Setup the Metric
+You may use any LLM based metric. For simplicity, I am using aspect critic metric and setting it up so that it can compare the response with the reference.
+
+
+```python
+critic = AspectCritic(name="answer_correctness",definition="Given the user_input, reference and response. Is the response correct compared with the reference",llm=llm)
+
+```
+
+### Evaluate and Upload the results
+
+
+```python
+results = evaluate(eval_dataset,metrics=[critic])
+
+```
+
+    Evaluating: 100%|██████████████████████████████████████████████████████████| 56/56 [00:01<00:00,  3.49it/s]
+
+
+
+```python
+results.upload()
+```
+
+Evaluation results uploaded! View at https://app.ragas.io/dashboard/alignment/evaluation/a6baf6ff-027f-4097-89e3-e11c70b8cf61
+'https://app.ragas.io/dashboard/alignment/evaluation/a6baf6ff-027f-4097-89e3-e11c70b8cf61'
+
+
+## Annotation
+
+### Review and annotate some results
+You may now view and annotate the evaluation results in app.ragas. These annotations will be used to train the metric. Please make sure to annotate at least 15-20 examples for good results.
+
+<figure markdown="span">
+  ![Annotating](../../../_static/imgs/annotation.png){width="600"}
+  <figcaption>Annotating results</figcaption>
+</figure>
+
+
+## Training and Alignment
+
+### Train the metric
+Download the annotated samples from app.ragas using `Download annotated json` button. 
+Instruction and demonstration configurations are required tells ragas how to optimize instruction and few shot demonstrations respectively. You can customize these configurations as per your requirements.
+
+```python
+from ragas.config import InstructionConfig,DemonstrationConfig
+demo_config = DemonstrationConfig(embedding = embeddings)
+inst_config = InstructionConfig(llm=llm)
+```
+
+
+```python
+critic.train(path="edited_chain_runs.json",demonstration_config=demo_config,instruction_config=inst_config)
+```
+
+    Fitness Evaluation Step 4/4: 100%|█████████████████████████████████████| 146/146 [00:24<00:00,  6.03it/s]
+    Few-shot examples [single_turn_aspect_critic_prompt]: 100%|██████████████| 18/18 [00:09<00:00,  1.82it/s]
+
+
+### Analysis
+Now, let's do some analysis on the trained metric.
+
+First, let's take a look at new instructions that was obtained for the metric after training.
+
+
+```python
+critic.get_prompts()['single_turn_aspect_critic_prompt'].instruction
+```
```

Comment:
make this a print ? it will render in docs better

also should we show the old prompt here, so users can easily compare?